addons/account/wizard/account_move_reversal_view.xml

```diff
@@ -28,15 +29,16 @@
                          </group>
                     </group>
                     <group>
-                         <group>
-                             <field name="reason" attrs="{'invisible': [('move_type', '=', 'entry')]}"/>
-                             <field name="date_mode" string="Reversal Date" widget="radio"/>
-                         </group>
-                         <group>
-                            <field name="journal_id"/>
-                            <field name="date" string="Refund Date" attrs="{'invisible': ['|', ('move_type', 'not in', ('out_invoice', 'in_invoice')), ('date_mode', '!=', 'custom')], 'required':[('date_mode', '=', 'custom')]}"/>
-                            <field name="date" attrs="{'invisible': ['|', ('move_type', 'in', ('out_invoice', 'in_invoice')), ('date_mode', '!=', 'custom')], 'required':[('date_mode', '=', 'custom')]}"/>
-                         </group>
+                        <group>
+                            <field name="move_type"/>
```

Comment:
you can remove the invisible `<field name="move_type"/>` above

Reply:
Added field for debug, forgot to remove it